.github/workflows/deploy-staging.yaml

```diff
@@ -117,3 +117,12 @@ jobs:
           target: tonomy_blockchain_nodejs
           push: true
           tags: tonomy/blockchain:nodejs-development
+
+      - name: Build and push
+        uses: docker/build-push-action@v4
+        with:
+          context: .
+          file: ./blockchain/Dockerfile
+          target: tonomy_blockchain_easycleos
+          push: true
+          tags: tonomy/blockchain:easycleos-development
```

Comment:
we need to have a build script for master, which deploys production tag
`tags: tonomy/blockchain:easycleos`